web/src/shared/helpers/auth-tools.ts

```diff
@@ -62,14 +63,26 @@ export const getCurrentTenant = (): string => localStorage.getItem(AUTH_CURRENT_
 
 export const refetchToken = async () => {
```

Comment:
I think after the fix we can remove that sentence we added in README about increasing of token lifespan (or we agreed to not do this, do you remember?).

Reply:
[PR](https://github.com/epam/badgerdoc/pull/719) is not approved, so I will just close it